bot/cogs/weather_cog.py

```diff
@@ -67,14 +67,14 @@ def getPageData(self, Lat, Lon, res_weather_json, city, is_cond, is_hr, is_day):
 
             # Building the Page
             page += f'Location: {city} ({Lat},{Lon})\n'
-            #page += f'Temperature: {round(temp,1)}°F\n'
-            page += f'Temperature: {temp}°F\n'
+            # page += f'Temperature: {round(temp,1)}°F\n'
+            page += f'Temperature: {temp}°F / {round((temp - 32) * (5 / 9), 2)}°C\n'
             page += f'Condition:\t{desc}\n\n'
 
-            page += f'Feels Like: {round(feels,1)}°F\n'
+            page += f'Feels Like: {round(feels, 1)}°F / {round((feels - 32) * (5 / 9), 2)}°C\n'
```

Comment:
Is there a reason behind rounding Fahrenheit to one decimal but Celcius to two?

Reply:
Yeah thats a typo, fixed it